src/main/java/ro/msg/learning/shop/utils/DTOMapper.java

```diff
@@ -0,0 +1,37 @@
+package ro.msg.learning.shop.utils;
+
+import ro.msg.learning.shop.dtos.ProductDTO;
+import ro.msg.learning.shop.models.Product;
+import ro.msg.learning.shop.models.ProductCategory;
+
+public class DTOMapper {
+    public static ProductDTO map(Product product) {
```

Comment:
Rename these methods to something more relevant, like mapToProductDTO , mapToProduct.